src/sagas/nanoContract.js

```diff
@@ -0,0 +1,89 @@
+import {
+  ncApi
+} from '@hathor/wallet-lib';
+import {
+  takeEvery,
+  select,
+  all,
+  put,
+  call,
+} from 'redux-saga/effects';
+import { t } from 'ttag';
+import { NanoRequest404Error } from '@hathor/wallet-lib/lib/errors';
+import { STORE } from '../store';
```

Comment:
We should delete this import. It's not used anymore

Reply:
Done at https://github.com/HathorNetwork/hathor-wallet-mobile/pull/427/commits/6f37713af9f1f784a6bbfa39f27698df78c9863f